site/src/components/layout/layout.md

```diff
@@ -58,7 +58,7 @@ The following code performs the required layout:
 {{{example-code}}}
 ```
 
-The flexbox layout will be computed, resulting in translate transforms being applied to each element. Group elements do not have width or height properties, so this information is added via `layout-width` and `layout-height` properties. The layout component also provides a convenience accessor for these values, e.g. `selection.layout('width')`.
+Nodes are positioned and sized appropriately using a combination of `x`, `y`, `width`, `height` and `transform` attributes depending on the whether the node is an `svg`, `rect` or other. Additionally `layout-width`, `layout-height`, `layout-x` and `layout-y` attributes are added which can be accessed using e.g. `selection.layout('width')`.
```

Comment:
Currently it adds `x` and `y` properties without the `layout-` prefix, which looks like a bug.